DAL/contentInspectionApi.js

```diff
@@ -31,10 +31,16 @@ function isRestrictedHeader(contentType) {
 
     contentType = contentType.toLowerCase();
 
-    if (contentType === "image/gif")
-        return true;
-
-    return false;
+    switch (contentType) {
```

Comment:
We need to review why we're making this change.  The original scope of this bot was gifs are not opt-in for viewing, whereas videos have to be clicked.